package.json

```diff
@@ -28,5 +28,8 @@
     "cleanup": "rm -rf android/build ios/build yarn.lock node_modules && cd examples && rm -rf android/build android/app/build ios/build yarn.lock node_modules",
     "cleanup:android": "cd examples/android && ./gradlew clean",
     "cleanup:install": "yarn cleanup && yarn && cd examples && yarn && cd .. && yarn symlink && yarn cleanup:android"
+  },
+  "dependencies": {
```

Comment:
What's the need for adding it here? the package itself doesn't use any prop-type.